app/src/main/res/layout/administrator_controls_activity.xml

```diff
@@ -25,9 +25,10 @@
     <fragment
       android:id="@+id/administrator_controls_activity_fragment_navigation_drawer"
       android:name="org.oppia.android.app.drawer.NavigationDrawerFragment"
-      android:layout_width="@dimen/navigation_drawer_width"
+      android:layout_width="wrap_content"
       android:layout_height="match_parent"
       android:layout_gravity="start"
+      android:minWidth="@dimen/navigation_drawer_width"
```

Comment:
Accessibility Scanner (will be written as A11YS from now) is passing without any change in this file

Reply:
Added screenshot for same in description